app/assets/css/main.styl

```diff
@@ -1,5 +1,8 @@
 @require 'variables'
 
+*
```

Comment:
This is the most specific selector that exists. Any style on the site that requires `content-box` on anything will automatically be overwritten by this. Dangerous